kubeval/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.8
```

Comment:
Hmm, is there any restriction against using newer versions?

Reply:
Works with 3.10, updated.

Reply:
I think we can unpin it, this is not a critical piece of software, and is not running anywhere exposed to the world. And it will lower the maintenance.

Reply:
OK